pkg/util/validation/validation.go

```diff
@@ -17,20 +17,28 @@ limitations under the License.
 package validation
 
 import (
+	"fmt"
 	"math"
 	"net"
 	"regexp"
+	"strconv"
 	"strings"
 )
 
 const qnameCharFmt string = "[A-Za-z0-9]"
 const qnameExtCharFmt string = "[-A-Za-z0-9_.]"
-const QualifiedNameFmt string = "(" + qnameCharFmt + qnameExtCharFmt + "*)?" + qnameCharFmt
-const QualifiedNameMaxLength int = 63
-
-var qualifiedNameRegexp = regexp.MustCompile("^" + QualifiedNameFmt + "$")
-
-func IsQualifiedName(value string) bool {
+const qualifiedNameFmt string = "(" + qnameCharFmt + qnameExtCharFmt + "*)?" + qnameCharFmt
```

Comment:
These were public because they are being used by downstream code not in this package to present consistent messages. and values.  I'm not sure hiding them still allows that - @deads2k